### DISCUSSION TITLE
Need Help Understanding

### DISCUSSION THREAD

**Discussion by Bandgeek1711** [inkstitch/inkstitch#1692] [Q&A]
Ok so I have been attempting to figure out how to satin stitch a logo that a friend is needing on their shirts for work. I have watched more than I can even comprehend of videos explaining how to do it and follow along fine but keep getting errors even though I am doing exactly what they are. Not sure if I just don't have everything installed properly or what but I'm to the point of going insane. Can anybody see or guide me to what I'm doing incorrectly?
Any and all advice is greatly appreciated!
![image](https://user-images.githubusercontent.com/107588540/173942984-1b4156bb-f602-4f55-874a-7b09f942555b.png)
![image](https://user-images.githubusercontent.com/107588540/173943100-a5223d32-6f45-44d5-83d0-fa4bd678b074.png)

**Comment by tyrosinase** [comment]
I have not watched any of the videos because videos are not my thing, so I can't really help other than to say if videos are also not your thing, there's an alternative tutorial: https://silverseams.com/tutorials/digitizing-with-ink-stitch (which has a link to a screen-by-screen walkthru, which might be a little dated as to version though).

**Reply by Bandgeek1711** [reply]
Thank you. I will take another look at that walkthru as well. I have kinda been learning this program as I go and have attempted to use a number of different resources and have just been coming up short on the satin stitch portion. Hopefully that walkthrough will help with the errors.

**Comment by buzzmandt** [comment]
I made a tutorial specifically on satin stitch fonts, from simple to very complex. I think it might help you here. After you have watched it I will try to answer any questions you might have.
[https://youtu.be/LFdQuHP5dRY](https://youtu.be/LFdQuHP5dRY)

**Reply by Bandgeek1711** [reply]
This was actually the video that I currently have pulled up that I was following. I also tried doing step by step what you did and keep getting errors in the param step. I'm still unsure if I possibly have something installed incorrectly.

**Comment by kaalleen** [comment]
![173942984-1b4156bb-f602-4f55-874a-7b09f942555b](https://user-images.githubusercontent.com/36401965/174001270-60530b74-b362-4543-bbac-32914ed72d71.png)

This is the problematic path. It has a fill and probably it doesn't belong to the paths that you want to use as a satin.

1. If it should be used as a fill: Run `Extension > Ink/Stitch > Fill Tools > Break apart fill objects...` to split it into multiple path elements if you want it to work (as a fill).

2. If you actually want to use this specific path as a satin column: Open te fill and stroke dialog (Object > Fill and stroke) and remove the fill.

When you run into troubles, the troubleshoot objects extension also can deliver helpful info on how to solve issues (info will be added beside of the canvas).

**Reply by Bandgeek1711** [reply]
Thank you so incredibly much for the information. I will try those suggestions later today!
